recipes/chopin2/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "chopin2" %}
+{% set version = "1.0.4" %}
+{% set sha256 = "8f9537e97ff9bd2e199dd6da39ba37d0076a385cb1187bef89da4fee12fd5911" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/fabio-cumbo/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: "{{ sha256 }}"
+
+build:
+  number: 0
+  skip: Ture  # [py<38 or win]
```

Comment:
Thanks for your review @ocefpaf 

I skipped Windows because I was struggling with this error: `[noarch_python] Noarch package contains binary script: chopin2.exe`

It is probably related to this [https://github.com/conda/conda-build/issues/2627](https://github.com/conda/conda-build/issues/2627), but I'm honestly not sure how to solve it.

Should I add the same python entry-points that I defined in the setup.py to the conda recipe?

Reply:
@ocefpaf problem solved.
Please let me know if you think it could be merged now. Thanks!